task_definitions/staging/task_difinition.json

```diff
@@ -0,0 +1,194 @@
+{
+  "taskDefinitionArn": "arn:aws:ecs:us-east-2:693260174422:task-definition/ette:1",
```

Comment:
file names are milspelt task_difinition should be definition. We only need this to run on staging and prod. Not in pre-prod to minimise our costs.